admin-ui/src/components/add-account-dialog.tsx

```
@@ -0,0 +1,496 @@
+    }
+    setRemainingSeconds(null)
+  }, [])
+
+  const startCountdown = useCallback(
+    (expiresInSeconds: number) => {
+      clearCountdown()
+      expiresAtRef.current = Date.now() + expiresInSeconds * 1000
+      setRemainingSeconds(expiresInSeconds)
+
+      countdownRef.current = setInterval(() => {
+        const remaining = Math.max(0, Math.ceil((expiresAtRef.current - Date.now()) / 1000))
```

ESLint

[error] Replace `0,·Math.ceil((expiresAtRef.current·-·Date.now())·/·1000)` with `⏎··········0,⏎··········Math.ceil((expiresAtRef.current·-·Date.now())·/·1000),⏎········`

(prettier/prettier)